terracumber_config/tf_files/SUSE-Manager-4.3-PR-tests-env5.tf

```diff
@@ -172,12 +173,12 @@ module "cucumber_testsuite" {
   auth_registry      = "registry.mgr.prv.suse.net:5000/cucutest"
   auth_registry_username = "cucutest"
   auth_registry_password = "cucusecret"
-  git_profiles_repo = "https://github.com/uyuni-project/uyuni.git#:testsuite/features/profiles/internal_prv"
+  git_profiles_repo = "https://github.com/SUSE/spacewalk.git//:testsuite/features/profiles/internal_prv"
```

Comment:
I would say we use Uyuni repo for both SUMA and Uyuni Docker profiles.
I think we don't need to change this line.